python/integrationtest/config/df-segment-1df-tpw.data.xml

```diff
@@ -0,0 +1,265 @@
+  >
+  <!ELEMENT obj (attr | rel)*>
+  <!ATTLIST obj
+      class CDATA #REQUIRED
+      id CDATA #REQUIRED
+  >
+  <!ELEMENT attr (data)*>
+  <!ATTLIST attr
+      name CDATA #REQUIRED
+      type (bool|s8|u8|s16|u16|s32|u32|s64|u64|float|double|date|time|string|uid|enum|class|-) "-"
+      val CDATA ""
+  >
+  <!ELEMENT data EMPTY>
+  <!ATTLIST data
+      val CDATA #REQUIRED
+  >
+  <!ELEMENT rel (ref)*>
+  <!ATTLIST rel
+      name CDATA #REQUIRED
+      class CDATA ""
+      id CDATA ""
+  >
+  <!ELEMENT ref EMPTY>
+  <!ATTLIST ref
+      class CDATA #REQUIRED
+      id CDATA #REQUIRED
+  >
+]>
+
+<oks-data>
+
+<info name="" type="" num-of-items="17" oks-format="data" oks-version="oks-08-03-04-11-g3f5bde7 built &quot;Feb 21 2023&quot;" created-by="gjc" created-on="thinkpad" creation-time="20231207T105629" last-modified-by="glehmann" last-modified-on="np04-srv-019.cern.ch" last-modification-time="20240325T124934"/>
+
+<include>
+ <file path="schema/coredal/dunedaq.schema.xml"/>
+ <file path="schema/appdal/application.schema.xml"/>
+ <file path="config/appdal/connections.data.xml"/>
+ <file path="config/appdal/moduleconfs.data.xml"/>
+  <file path="config/appdal/fsm.data.xml"/>
+</include>
+
+<comments>
+ <comment creation-time="20231207T111607" created-by="gjc" created-on="thinkpad" author="gjc" text="u"/>
+ <comment creation-time="20231207T111909" created-by="gjc" created-on="thinkpad" author="gjc" text="h"/>
+ <comment creation-time="20240112T145841" created-by="eflumerf" created-on="ironvirt9.mshome.net" author="eflumerf" text="Adding storage"/>
+ <comment creation-time="20240112T163053" created-by="eflumerf" created-on="ironvirt9.mshome.net" author="eflumerf" text="Add TRB"/>
+ <comment creation-time="20240112T170242" created-by="eflumerf" created-on="ironvirt9.mshome.net" author="eflumerf" text="Add queue"/>
+ <comment creation-time="20240112T170541" created-by="eflumerf" created-on="ironvirt9.mshome.net" author="eflumerf" text="Add network connections"/>
+ <comment creation-time="20240208T092804" created-by="eflumerf" created-on="ironvirt9.mshome.net" author="eflumerf" text="reorder"/>
+ <comment creation-time="20240220T094213" created-by="glehmann" created-on="np04-srv-024.cern.ch" author="Unknown" text="updated configuration to use a dedicated app for TA making"/>
+ <comment creation-time="20240306T200127" created-by="glehmann" created-on="np04-srv-024.cern.ch" author="Unknown" text="added TP stream writer"/>
+ <comment creation-time="20240307T084519" created-by="glehmann" created-on="np04-srv-024.cern.ch" author="Unknown" text="added dataflow"/>
+ <comment creation-time="20240312T202726" created-by="eflumerf" created-on="ironvirt9.mshome.net" author="eflumerf" text="Trivial edit to reorder objects"/>
+ <comment creation-time="20240325T124934" created-by="glehmann" created-on="np04-srv-019.cern.ch" author="Unknown" text="adjusted to name changes in connections"/>
+</comments>
+
+
+
+  <obj class="PhysicalHost" id="localhost">
```

Comment:
Sure host info should be defined separately from the segment. What happens if you include two segments that define localhost in the same session?

Reply:
Since the ids are the same, my experience is that they get squashed down to 1 instance in consolidate.py